crates/net/network/src/lib.rs

```diff
@@ -58,7 +58,7 @@ mod fetch;
 mod import;
 mod listener;
 mod manager;
-mod message;
+pub mod message;
```

Comment:
after thinking about this a bit more, I'd rather like to pub use `PeerRequest` instead of pub mod message here

Reply:
Yeah that makes more sense, it looks like it's the only struct in the module that actually needs to be public